rearrangement_benchmark/tasks/move_task.py

```diff
@@ -0,0 +1,4 @@
+from .rearrangement_task import RearrangementTask
```

Comment:
I think we should generally avoid relative imports.
```suggestion
from rearrangement_benchmark.rearrangement_task import RearrangementTask
```